hellofresh-py3-test.yml

```
@@ -0,0 +1,9 @@
+name: hellofresh-py3-test
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.7
+  - pip:
+    - --index-url https://artifactory.tools-k8s.hellofresh.io/artifactory/api/pypi/py/simple
+    - biautomation==1.1.0
```

```suggestion
    - biautomation>=2.0.0,<3.0.0
```

Grid Dynamics migrated biautomation to work on Python 3 [here](https://github.com/hellofresh/bi-automation/pull/29) (PR still open, but pack manually published to Artifactory).